package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jitera/jitera-web-ui-library",
-  "version": "1.4.2",
+  "version": "1.4.3",
```

Comment:
no need to change this, it acutally 1.7.x

Reply:
@duongjitera i will update